assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -14,16 +16,33 @@ import './editor.scss';
 
 const CheckoutEditor = ( { attributes } ) => {
 	const { className } = attributes;
-	// @todo: wrap Block with Disabled once you finish building the form
 	return (
 		<div className={ className }>
-			<Block
-				attributes={ attributes }
-				isEditor={ true }
-				shippingRates={
-					SHIPPING_METHODS_EXIST ? previewShippingRates : []
-				}
-			/>
+			<BlockErrorBoundary
+				header={ __(
+					'Checkout Block Error',
+					'woo-gutenberg-products-block'
+				) }
+				text={ __(
+					'There was an error whilst rendering the checkout block. If this problem continues, try re-creating the block.',
+					'woo-gutenberg-products-block'
+				) }
+				showErrorMessage={ true }
+				errorMessagePrefix={ __(
+					'Error message:',
+					'woo-gutenberg-products-block'
+				) }
+			>
+				<Disabled>
```

Comment:
I was thinking about this today. We can't really wrap the entire block with `<Disabled>` because when there are no shipping methods there is a button we want to be clickable:
![imatge](https://user-images.githubusercontent.com/3616980/75796360-d25aa700-5d73-11ea-8bd3-ac62d1012b42.png)

I think the only solution for that would be to dynamically wrap each part of the form with `<Disabled>`.